bin/start

```diff
@@ -1,22 +1,36 @@
 #!/usr/bin/env bash
 
+if [ -f .env ]; then
+  export $(echo $(cat .env | sed 's/#.*//g'| xargs) | envsubst)
+fi
+
 PORT="${PORT:=3000}"
-PGHOST=postgres
-PGPORT=5432
-RSHOST=redis
-RSPORT=6379
 
-echo "Greenlight-v3 starting on port: $PORT"
+# Parse Rails DATABASE and REDIS urls to get host and port
+pgProtocol="$(echo $DATABASE_URL | grep :// | sed -e's,^\(.*://\).*,\1,g')" # extract the protocol
+pgUrl="$(echo ${DATABASE_URL/$pgProtocol/})" # remove the protocol
+pgUser="$(echo $pgUrl | grep @ | cut -d@ -f1)" # extract the user (if any)
+pgHostport="$(echo ${pgUrl/$pgUser@/} | cut -d/ -f1)" # extract the host and port
+pgHost="$(echo $pgHostport | sed -e 's,:.*,,g')" # by request host without port
```

Comment:
I think all of this logic is optimistic,
Therefore, this can be simplified to something like `pgHost="$(echo $pgHostport | cut -d: -f1)" # by request host without port`